src/utils/utils.ts

```diff
@@ -15,22 +17,22 @@ export class Utils {
     const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex);
 
     return result ? {
-      r: parseInt(result[1], 16),
-      g: parseInt(result[2], 16),
       b: parseInt(result[3], 16),
+      g: parseInt(result[2], 16),
+      r: parseInt(result[1], 16),
     } : null;
   }
 
-  public static clamp(number: number, min: number, max: number) {
-    return Math.min(Math.max(number, min), max);
+  public static clamp(num: number, min: number, max: number): number {
+    return Math.min(Math.max(num, min), max);
   }
 
-  public static isInArray<T>(value: T, array: T[] | T) {
+  public static isInArray<T>(value: T, array: T[] | T): boolean {
     return value === array || (array as T[]).indexOf(value) > -1;
   }
 
-  public static deepExtend(destination: any, source: any) {
-    for (let property in source) {
+  public static deepExtend(destination: any, source: any): any {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)